hazelcast/src/test/java/com/hazelcast/internal/metrics/metricsets/OperatingSystemMetricSetTest.java

```diff
@@ -62,53 +64,43 @@ public void testGenericSensors() {
     }
 
     @Test
-    public void testComSunManagementUnixOperatingSystem() {
-        assumeOperatingSystemMXBean("com.sun.management.UnixOperatingSystem");
-
-        assertContainsSensor("os.committedVirtualMemorySize");
```

Comment:
Why has this been removed??

Reply:
It's already tested in `testComSunManagementOperatingSystemMXBean` method.
The assumptions check for interface implemented by the MX Bean.
The interface inheritance tree looks like:
```
└── java.lang.management.OperatingSystemMXBean
  └── com.sun.management.OperatingSystemMXBean
    └── com.sun.management.UnixOperatingSystemMXBean
```